ocs_ci/ocs/ui/page_objects/buckets_tab.py

```
@@ -632,6 +636,16 @@ def upload_folder_to_bucket(self, folder_path: str, wait_time: int = 2) -> None:
         file_input.clear()
         file_input.send_keys(folder_path)
         time.sleep(wait_time)
+        folder_name = os.path.basename(folder_path)
+        try:
+            self.wait_for_element_to_be_present(
+                format_locator(self.bucket_tab["uploaded_folder_name"], folder_name),
+            )
+            logger.info(f"Upload verified: {folder_name} appears in bucket")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)